etc/testing/passing_test_regex.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# This script prints a regex (which should be passed to 'go test -run=<regex>')
+# that matches all tests in the pachyderm test suite that aren't in
+# etc/testing/blacklist.txt. Tests in blacklist.txt are flaky or otherwise
+# cause our test suite to consitently flake or fail when run in a cloud
+# provider's cluster and would render our nightly tests useless.
+
+basedir="$(realpath "$(dirname "${0}")/../..")"
+tmpfile="$(mktemp --dry-run --tmpdir=.)"
+
+# Put blacklisted tests in ${tmpfile}
+grep -v '^$' "${basedir}/etc/testing/blacklist.txt" >"${tmpfile}"
+
+# List all tests in src/server, and put them in ${tmpfile} as well
+ls "${basedir}/src/server" \
+  | grep '_test\.go$' \
+  | while read f; do
+      grep --no-filename '^func Test[A-Za-z_]\+(.* \*testing\.T)' "${basedir}/src/server/${f}" \
```

Comment:
So to be clear -- it is our intent to run only the stuff directly under `src/server` ... basically the integration tests? I think that makes sense because the integration tests are the ones most likely to use / test the cloud deployment. I just want to be clear on our intent.